WordPress/Classes/ViewRelated/System/WPTabBarController+Swift.swift

```diff
@@ -84,7 +84,13 @@ extension WPTabBarController {
                 return false
         }
 
-        WPAppAnalytics.track(stat)
+        if tabType == .reader {
+            let variation = ABTest.biasTest.variation == .control ? "control" : "treatment"
+            WPAppAnalytics.track(stat, withProperties: ["experiment_id": 20086, "variation": variation])
```

Comment:
The event with extra props should only fire when the API returns a variation. If it fails to retrieve an experiment assignment, no extra props should be returned even if the `tabType == .reader`.

Reply:
@aaronyan Knowing if an experiment "exists" is something that we should worry about or is it specific for this case?

We used to have an `unknown` property for events that didn't exist (or expired), but we changed the implementation to always return `control` in this case ([see here](https://github.com/Automattic/Automattic-Tracks-iOS/pull/155#discussion_r560243072)).

Reply:
Yeah, we don't need to know this anymore. We've decided to drop the eventprop requirement, so I'm going to mark this thread as resolved!